test/scrabble_test.js

```diff
@@ -0,0 +1,14 @@
+const expect = require('chai').expect
+const scrabble = require('../scripts/scrabble.js')
+
+describe( "Scrabble", () => {
+
+  it( "getInput() will return an array", () => {
+    expect( getInput() ).to.be.instanceof(Array)
+  })
+
+  it( "bagCheck() is an object", () => {
```

Comment:
I don't understand the intent of this test, and because of this, I don't understand the intent of the `bagCheck` function.  I'm guessing that we're asking if a letter is allowed? exists in the collection, or has been played already?  I would consider both a new description for this test that clearly articulates the purpose of the test - what are you validating and why - as well as a new name for the `bagCheck` function that explicitly identifies its purpose.